src/OSPSuite.Assets/UIConstants.cs

```diff
@@ -1210,6 +1211,7 @@ public static class Error
       public static readonly string InavlidErrorDimension = "The dimension of the error units must be the same as the dimension of the measurement units.";
       public static readonly string NaNOnData = "Data contains NaN values at imported columns. Select a different action for NaN values or clean your data.";
       public static readonly string UnsupportedFileFormat = "The file format is not supported";
+      public static readonly string InconsistenMoleculeAndMoleWeightException = "Molecule and Molecular Weight does not match. Please either edit your Molecule-Molecular Weight in your project or remove the Molecular Weight from your mappings";
```

Comment:
DO NOT MATCH

What does `Molecule-Molecule Weight` means?

Reply:
@msevestre Changed to "Molecule and Molecular Weight do not match. Please either edit your Molecule or your Molecular Weight in your project or remove the Molecular Weight from your mappings"